spring-data-r2dbc/src/main/java/org/springframework/data/r2dbc/core/ReactiveSelectOperation.java

```diff
@@ -115,6 +117,14 @@ interface SelectWithProjection<T> extends SelectWithQuery<T> {
 	 */
 	interface SelectWithQuery<T> extends TerminatingSelect<T> {
 
+		/**
+		 * Specifies the fetch size for this query
+		 *
+		 * @param fetchSize
+		 * @return
+		 */
+		SelectWithQuery<T> withFetchSize(Integer fetchSize);
```

Comment:
this should be a primitive.

Reply:
I agree

Reply:
@mp911de I hope we both understand, that in general, this does not omit the boxing from `int` to `Integer`. Although, specifying `withFetchSize(null)` still does not make much sense, since that signals the driver to adhere to default.

Reply:
We apply defaults whenever user input is absent. Asking for applying default behavior is weird in the first place.

Reply:
That's exactly what I'm trying to communicate, I'm just verifying we're on the same page.